build.gradle.kts

```diff
@@ -5,7 +5,11 @@ buildscript {
     repositories {
         google()
         mavenCentral()
-        jcenter()
+        jcenter {
+            content {
+                includeGroup("org.jetbrains.kotlinx")
```

Comment:
Required for HTML reports. We might also use kotlinx for introducing truly immutable lists in the data classes again, to Java safety.

Reply:
Is that lib released? Immutable collections I mean.

Reply:
Hm, I've just read up on it [here](https://www.baeldung.com/kotlin/immutable-collections), without checking the release status. Seems it's still in pre-release.